Framework/DataHandling/test/LoadPSIMuonBinTest.h

```diff
@@ -209,6 +211,64 @@ class LoadPSIMuonBinTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove("ws");
   }
 
+  void test_time_zero_table_loaded_correctly() {
+    LoadPSIMuonBin alg;
+    alg.initialize();
+    alg.isInitialized();
```

Comment:
Your not asserting this, it likely can be removed though as if `init` fails every unit will fall over with the same error

Alternatively, if you want to keep it create a new test called
```
void test_alg_can_init(){
   LoadPSIMuonBin alg;
   alg.initialize();
   TS_ASSERT(alg.isInitialized());
}
```
That way if the test fails it says quite clearly `LoadPSIMuonBinTest.test_alg_can_init FAILED`

Reply:
Good spot. should be covered by the init test at the top of the file